src/Framework/TestResult.php

```diff
@@ -22,6 +22,11 @@
 class PHPUnit_Framework_TestResult implements Countable
 {
     /**
+     * @var integer
+     */
+    protected $percentageSkip = 0;
```

Comment:
Please make this `private`. Potential child classes can use the `public` accessor and mutator methods the patch also introduces.